src/sentry/monitors/utils.py

```diff
@@ -313,8 +315,21 @@ def update_alert_rule(request: Request, project: Project, alert_rule: Rule, aler
             "project": project,
             "actions": data.get("actions", []),
             "environment": data.get("environment", None),
-            # TODO(davidenwang): This is kind of a hack to get around updater removing conditions if not passed
-            "conditions": alert_rule.data.get("conditions", []),
+            "name": f"Monitor Alert: {monitor.name}"[:64],
+            "conditions": [
```

Comment:
is this always going to revert the conditions back to these set of 3 if the user edits the monitor?

Reply:
yeah but i don't want to get too deep in the weeds manually parsing the conditions (and updating the slug there). IMO better to just overwrite. this alerting isn't a long term solution either

Reply:
i can loop through conditions though if necessary

Reply:
yea that's fair it's definitely annoying to manually loop/parse the conditions, but i do think customers have changed their conditions for their own use case and it would be unexpected for a name change to reset those changes. I think looping thru to just edit the one event tag condition would be good.

Reply:
slug updates are pretty rare thankfully lol